sleap/gui/app.py

```diff
@@ -1236,6 +1238,12 @@ def _has_topic(topic_list):
         if _has_topic([UpdateTopic.frame, UpdateTopic.project_instances]):
             self.state["last_interacted_frame"] = self.state["labeled_frame"]
 
+        if _has_topic([UpdateTopic.sessions]):
+            self.sessions_dock.table["camera_table"].model().items = self.labels.sessions[0]
```

Comment:
On some of the tests we get an `IndexError` when trying to index `labels.sessions[0]` since not all `Labels` projects will have at least one `RecordingSession` in `labels.sessions: List[RecordingSession]`. So, we cannot use a hard-coded 0-index. Furthermore, I think we would want to use whichever `RecordingSession` is loaded in the `GuiState` (in `self.state["session"]`) here not `self.labels.sessions[0]`.